src/main/java/com/exasol/adapter/dynamodb/mapping/IterationIndexColumnMapping.java

```diff
@@ -9,9 +9,8 @@
  * This class defines a column that maps the array index of a nested list. Such columns are useful for nested tables
  * that do not have an key.
  */
-public class IterationIndexColumnMapping implements ColumnMapping {
+public class IterationIndexColumnMapping extends AbstractColumnMapping {
```

Comment:
I am not sure about this abstract class. It does not contain abstract methods. 

Suggestion, can you make it `BaseColumnMapping` or `SimpleColumnMapping` and not extend it in other classes? Let other classes implement interfaces.

Reply:
It will also reduce super method calls in equals and hashcode methods.

Reply:
I added this abstract class to reduce code dublication on the getExasolColumnName getter. I think it quite nicew that i can delegate the equal call to super?